webpack/components/Search/index.js

```diff
@@ -33,13 +33,19 @@ class Search extends Component {
       params.params || {},
     ];
 
-    api.get(...autoCompleteParams).then(({ data }) => {
-      this.setState({
-        items: data.filter(({ error }) => !error).map(({ label }) => ({
-          text: label.trim(),
-        })),
+    api
+      .get(...autoCompleteParams)
+      .then(({ data }) => {
+        this.setState({
+          items: data.filter(({ error }) => !error).map(({ label }) => ({
+            text: label.trim(),
+          })),
+        });
+      })
+      .catch((error) => {
+        // eslint-disable-next-line no-console
+        console.log(error);
```

Comment:
Why do we want to `console.log()` this error?  We should do something useful with the error or do nothing at all.

Reply:
@waldenraines This is the question I asked @tbrisker [here](https://github.com/Katello/katello/pull/7555#discussion_r204158046), maybe he can expand on it? (the comment was collapsed so it was hidden).

Reply:
@waldenraines The reason I added the catch block is otherwise node throws a "unhandled promise" warning, but running this again, I see this is only because I was using a newer version of nodejs. I completely removed the catch block without any warnings, I think this is fine for now (until we update node versions)